retriever/__main__.py

```diff
@@ -185,6 +192,26 @@ def main():
                             )
                         )
                         count += 1
+                    print("Available online datasets : {}\n".format(len(searched_scripts['online'])))
+                    count = 1
+                    searched_scripts_offline = [script.name for script in searched_scripts['offline']]
+                    for script in searched_scripts['online']:
+                        if script in searched_scripts_offline:
+                            continue
+                        print("{count}. {name}".format(
```

Comment:
https://github.com/weecology/retriever/pull/1374/files#diff-114dd1262f915d1698d5214cc07ee396R36
https://github.com/weecology/retriever/pull/1374/files#diff-114dd1262f915d1698d5214cc07ee396R201

We have some hanging indentation in these two places

```python
        if args.command not in ['reset', 'update'] \
        and not os.path.isdir(SCRIPT_SEARCH_PATHS[1]) \
        and not [f for f in os.listdir(SCRIPT_SEARCH_PATHS[-1])
                 if os.path.exists(SCRIPT_SEARCH_PATHS[-1])]:
            check_for_updates()
            reload_scripts()
        script_list = SCRIPT_LIST()
```

```python
                    print("{count}. {name}".format(
                        count=count,
                        name=script
                    )
```